Crossref Unixref XML.js

```diff
@@ -0,0 +1,707 @@
+{
+	"translatorID": "93514073-b541-4e02-9180-c36d2f3bb401",
```

Comment:
Is this not a new version of the previous Crossref translator? If so, the id should remain the same.

(The previous one is also called from from arXiv.org translator.)

Reply:
I guess not, if that was a search translator and this is an import translator. The arXiv translator will need to be fixed, in any case.